bcgov/moh-prime
# PRIME-1075 SOAP Integration

How to test in Postman:

* POST `http://localhost:5000/api/PLRHL7`
* Set the Body to raw XML
* Ensure the Content-Type in the Header `application/xml` sometimes changing the Body to raw XML doesn't update the Content-Type from text

Body of POST:

```xml
<?xml version="1.0" encoding="utf-8"?>
<soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
    <soap:Body>
        <PRPM_IN301030CA ITSVersion="XML_1.0" xmlns="urn:hl7-org:v3">
            <Customers>
                <Customer customerId="1">
                    <FirstName>Tide</FirstName>
                    <LastName>Pods</LastName>
                    <Email>tide@pods.com</Email>
                </Customer>
            </Customers>
        </PRPM_IN301030CA>
    </soap:Body>
</soap:Envelope>
```

Should write out to console everything within the `<soap:Body>` element
